provision/provision-site.sh

```diff
@@ -76,6 +76,11 @@ function vvv_provision_site_nginx() {
   sed -i "s#{vvv_path_to_site}#${VM_DIR}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
   sed -i "s#{vvv_site_name}#${SITE}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
   sed -i "s#{vvv_hosts}#${VVV_HOSTS}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
+
+  if [ 'php' != "${NGINX_UPSTREAM}" ] && [ ! -f "/etc/nginx/upstreams/${NGINX_UPSTREAM}.conf" ]; then
+    echo -e "${YELLOW} * Upstream ${NGINX_UPSTREAM} not found. Defaulting to php.${CRESET}"
+    NGINX_UPSTREAM='php'
```

Comment:
There is no `php` upstream, instead it should default to `php72` as that's what VVV core installs. We should also print out a list of the valid nginx upstream values

Reply:
There is a php upstream. Checked it personally.

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/develop/config/nginx-config/nginx.conf#L88-L91

Reply:
It's also the default upstream https://github.com/Varying-Vagrant-Vagrants/VVV/blob/develop/Vagrantfile#L189